README.md

```diff
@@ -37,24 +37,19 @@ deleteJarsTask := {
 ```
 - All your settings can be put inside a config.yaml file like below(Applicable only for shell script).
 ```
-ORGANIZATION : <orgnaziation docker hub name>
```

Comment:
Not valid YAML, missing quotes for some strings, quotes used for integers, comma missing in object rules, spaces missing after : ,
inconsistent casing of vars , VERSION should be version
should be formatted like this:

```
dockerusername: "zeidkleo"
version: "latest"
databricks_profile: "dev"
roles: {
  instance_profile_arn: "arn:aws:iam::853628181016:instance-profile/ETL_Databricks_Development_Role",
  assume_role_arn: "",
}
zone_id: "eu-west1b"
min_workers: 2
max_workers: 8
node_type_id: "i3.2xlarge"
autotermination_minutes: 120
driver_node_type_id: "i3.2xlarge"

```

Reply:
Will do required change and remove version

Reply:
Donot merge this one yet .I will do required changes here

Reply:
Done all required changes and added optimisation and now check status of each command before proceeding ahead